tests/python/shared/utils/resource_import_export.py

```diff
@@ -82,50 +82,79 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     def _export_resource_to_cloud_storage(
-        self, obj_id: int, obj: str, resource: str, *, user: str, **kwargs
+        self,
+        obj_id: int,
+        obj: str,
+        resource: str,
+        *,
+        user: str,
+        _expect_status: Optional[int] = None,
```

Comment:
```suggestion
        _expect_status: int = HTTPStatus.OK,
```

Also, why is this underscore-prefixed?

Reply:
This is prefixed to avoid confusion with normal method parameters, which are passed further into the request.